### ISSUE TITLE
Autocomplete enhancements fall 2022

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#2489]
Epic for issues related to `Autocomplete`.

In short, these revolve around 3 enhancements;
- [x] Keep search value on select
- [ ] `Chip` inside next to input
  - This might be solved inadvertently by #2336 with exposing `leftAdorment` prop
  - On problem might be that compact mode uses height, so this might be break multiple chips which need to grow
- [x] Expand `optionLabel` to support `ReactNode` instead of just strings

**Comment by TormodNy** [comment]
What is the status on these enhancements? In our project we have multiselect `Autocomplete` with many options, so the search is frequently used to search for the options needed. Often times the users want to select multiple options that have similar names and they have requested functionality for keeping the search value after an option is selected so that they can continue selecting without having to enter the same search query over again.

**Comment by oddvernes** [comment]
> What is the status on these enhancements? In our project we have multiselect `Autocomplete` with many options, so the search is frequently used to search for the options needed. Often times the users want to select multiple options that have similar names and they have requested functionality for keeping the search value after an option is selected so that they can continue selecting without having to enter the same search query over again.

We do have a `clearSearchOnChange={false}` to keep the search value and the dropdown open while selecting, is this what you are asking about?

**Comment by TormodNy** [comment]
Yes! I must have missed it when checking the storybook 🙃  Thank you!

**Comment by BirteThornquist** [comment]
This will also require design work.
